surveys/2021-annual-survey/questions.md

```diff
@@ -725,19 +725,27 @@ Select one:
 >
 > Would be useful for leadership to understand the community's fears.
 
-### In your opinion, how easy is it for qualified applicants to find a Rust job?
+### How much do you agree with the following statements on Rust employment?
+
+Statements:
+
+- It is easy for qualified applicants to find jobs which uses Rust for the majority of programming
+- Existing Rust jobs are attractive
+- Rust programming skills are in high demand
+- Rust provides me with skills that employers seek
```

Comment:
These last two seem the same to me

Reply:
I would also add something like "I feel qualified to apply for some advertised Rust jobs" or similar, to try and gauge if there are Rust jobs available but people don't think they can get them because they are too senior or domain-specific, etc.

Reply:
So I think they are quite different, but I'm not phrasing them well. The first is specifically about demand for Rust knowledge while the second is Rust providing the respondent with additional skills (e.g., systems programming knowledge, ability to work with advanced type systems, etc.) that are also employable.